rules/template-indent.js

```diff
@@ -67,9 +67,11 @@ const create = context => {
 		}
 
 		const dedented = stripIndent(joined);
+		const trimmed = dedented.replace(new RegExp(`^${eol}|${eol}[ \t]*$`, 'g'), '');
```

Comment:
I don't think this is correct.

It should only remove trailing empty new lines.

Reply:
@fisker This is the way it is already working and I was not sure if we should alter the existing behavior because it would be a breaking change

Reply:
Somehow I missed space in the regexp, it's good.